SPECS/apr/apr.spec

```diff
@@ -37,6 +37,7 @@ make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
+find %{buildroot} -type f -name "*.la" -delete -print
```

Comment:
Since it's always the same command, what do you think about adding a macro for it and calling it instead? This would make it less error-prone and the macro name would give an additional explanation of what we're doing.

Also, do you know, if there's a way to modify the default behaviour of `%install`? I'm curious, if we could get rid of the need to write this command everywhere. We could have an optional argument for `%install` for exceptional cases, where we might want to retain the libtool archives. Not sure, if that's possible, though.

Reply:
Looked at messing with the core macros a while ago. It IS possible to override them in a spec file, but probably not a good idea. If I recall correctly rpm defines a few `%<stage>_post` macros, might look into those? Its been a while since I looked though.

Reply:
The `%<stage>_post` macros sound promising. Btw. everything I've mentioned is a suggestion, we can do these potential improvements at a later time, if it's more than trivial to do.

Reply:
This is a great idea, but it would be duplicating work that the RPM maintainers have already done. A buildroot policy script that will handle libtool archive file removal by default was added in RPM 4.17.0, which we should be upgrading to on this branch.

Reply:
For reference: https://github.com/rpm-software-management/rpm/pull/1674